packages/core-data/src/types/user.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Internal dependencies
+ */
+import { AvatarUrls, Context, ContextualField } from './common';
+
+export interface User< C extends Context > {
+	/**
+	 * Unique identifier for the user.
+	 */
+	id: number;
+	/**
+	 * Login name for the user.
+	 */
+	username: ContextualField< string, 'edit', C >;
+	/**
+	 * Display name for the user.
+	 */
+	name: string;
+	/**
+	 * First name for the user.
+	 */
+	first_name: ContextualField< string, 'edit', C >;
+	/**
+	 * Last name for the user.
+	 */
+	last_name: ContextualField< string, 'edit', C >;
+	/**
+	 * The email address for the user.
+	 */
+	email: ContextualField< string, 'edit', C >;
+	/**
+	 * URL of the user.
+	 */
+	url: string;
+	/**
+	 * Description of the user.
+	 */
+	description: string;
+	/**
+	 * Author URL of the user.
+	 */
+	link: string;
+	/**
+	 * Locale for the user.
+	 */
+	locale: ContextualField< '' | 'en_US', 'edit', C >;
```

Comment:
is `en_US` the only possible locale value? we could never find `es_ES` or `cx_PH`?

Reply:
until then let's just leave it as `string` then to avoid confusion. we don't have to track the JSON schema bug since we're wanting to describe the API data, not the schema.